pkg/analysis/notimestamp/testdata/src/a/a.go

```diff
@@ -8,36 +8,36 @@ import (
 
 type NoTimeStampTestStruct struct {
 	// +optional
-	TimeStamp *time.Time `json:"timeStamp,omitempty"` // want "field TimeStamp: prefer use of the term time over timestamp"
```

Comment:
When we don't have `field <>` in the name, it makes exceptions hard to craft in the golangci-lint file, I think ideally we would restore this part